plenum/server/node.py

```diff
@@ -2412,13 +2423,113 @@ def doStaticValidation(self, request: Request):
 
         self.execute_hook(NodeHooks.POST_STATIC_VALIDATION, request=request)
 
-    def doDynamicValidation(self, request: Request):
+    def validateTaaAcceptance(self, request: Request, req_pp_time: int):
+        ledger_id = self.ledger_id_for_request(request)
+
+        if not self.ledgerManager.ledger_info(ledger_id).taa_acceptance_required:
+            logger.trace(
+                "{} TAA acceptance passed for request {}: "
+                "not required for ledger id {}"
+                .format(self, request.reqId, ledger_id)
+            )
+            return
+
+        config_req_handler = self.get_req_handler(ledger_id=CONFIG_LEDGER_ID)
+        if not config_req_handler:
+            raise LogicError("Config request handler is missed")
+
+        taa = None
+        taa_data = config_req_handler.get_taa_data()
+        if taa_data is not None:
+            (taa, taa_seq_no, taa_txn_time), taa_digest = taa_data
+
+        if taa is None:
```

Comment:
After a discussion with Andrey, we decided that no TAA must be present in a request if TAA is not set/disabled.